src/common/pmix_iof.c

```diff
@@ -1215,6 +1223,21 @@ void pmix_iof_read_local_handler(int unusedfd, short event, void *cbdata)
         goto reactivate;
     }
 
+    /* if I am a server, then push this up to my host */
+    if (PMIX_PROC_IS_SERVER(&pmix_globals.mypeer->proc_type)) {
+        if (NULL == pmix_host_server.push_stdin) {
+            /* nothing we can do with this info */
```

Comment:
```suggestion
            /* nothing we can do with this info */
            goto reactivate;
```
This seems to fix the IOF issue in CI.

Reply:
Ah...I had forgotten that our simple test servers don't support that function - never occurred to me to look at that path. Thanks!